src/components/axes/categoryAxis.ts

```diff
@@ -135,13 +129,15 @@ export module Axes {
      * @param {string[]} ticks The strings that will be printed on the ticks.
      */
     private _measureTicks(axisWidth: number, axisHeight: number, scale: Scales.Category, ticks: string[]) {
+      var axisSpace = this._isHorizontal() ? axisWidth : axisHeight;
+      var expectedRangeBand = axisSpace / (2 * scale.outerPadding() + (ticks.length - 1) * scale.innerPadding() + ticks.length);
```

Comment:
Why is this something that is expected? Should it be just rangeBand? Also, I can't figure out what is going on over here, which means the math is not well explained / not well encapsulated. I had to take pen and paper and still can't figure out if this is correct actually.